README.md

```diff
@@ -306,6 +306,8 @@ The available *options* are:
 * *pattern* - the [pattern](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern) attribute.
 * *minlength* - [minimum length](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/minlength) attribute.
 * *maxlength* - [maximum length](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength) attribute.
+* *max* - [maximum date (yyyy-mm-dd)](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max)
+* *min* - [minimum date (yyyy-mm-dd)](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/min)
```

Comment:
Are these values necessarily yyyy-mm-dd? From reading https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max it seems that it will depend on the input type ("month" will require max=yyyy-mm, etc). The "date" and "month" examples could be added after "email".

Reply:
Yes I agree. Since the date is probably the most common case, and the people often get the format wrong, I just left the link to the html docs and added a note about the format for date min and max